lona/static_file_loader.py

```diff
@@ -1,9 +1,20 @@
-from typing import TYPE_CHECKING, Optional, TypeVar, Type, List, Set
```

Comment:
I don't understand why this is necessary. I understand why to use dataclasses, the rest looks like it makes things nore complicated

Reply:
At the first sight it may seem more complicated, but I don't think so.

Actually there are two different objects: declaration of the file and real file (with correct attributes).
The same class was used in both cases. So it was necessary to `copy` and mutate attributes.
That was very unobvious and hidden deep inside the code.

Now they are explicitly split. And it is clear which class has which area of responsibility. And which class has which attributes.

Reply:
Ahhhh! Now i get it. Some explanation like this comment should be in the commit message.

Reply:
Agree, added explanation to commit message.

Reply:
I thought a lot about that and reread the code. I am sorry but i don't like this approach. From a plain modeling standpoint it makes sense, but it is a new abstraction with new types, new helper functions etc, to save one ``copy()``. Also ``StaticFile`` is very basic type. It's basically a dict with a nice name, so user code gets more verbose. I think, it's not worth the complexity.

I like the use of ``dataclass``, so i want to keep this. And i agree, it can be confusing that this copy is necessary, but i would want to fix that with a simple comment

Reply:
Ok, I'll merge them back.

Note: user's code will not change at all. There are no breaking changes.

Reply:
Done